lib/json_web_token/algorithm/rsa.ex

```diff
@@ -44,11 +43,14 @@ defmodule JsonWebToken.Algorithm.Rsa do
   end
 
   @doc "RSA key modulus, n"
-  def modulus(key) do
-    case @otp_vsn do
-      '20' -> :ssh_bits.mpint(Enum.at key, 1)
-      _ -> :crypto.mpint(Enum.at key, 1)
-    end
+  otp_vsn = :erlang.system_info(:otp_release)
+  |> to_string
+  |> String.to_integer
+
```

Comment:
Just my 2 cents, but this might be a good place for a comment explaining why we are doing this? 

"In version 20 of Erlang this function was moved to another module.... backwards compatibility.... "